zk-token-sdk/src/curve25519/ristretto.rs

```diff
@@ -64,6 +64,7 @@ mod target_arch {
 
         fn try_from(pod: &PodRistrettoPoint) -> Result<Self, Self::Error> {
             CompressedRistretto::from_slice(&pod.0)
+                .map_err(|_| Curve25519Error::PodConversion)?
```

Comment:
It is good to omit the error and returns `Curve25519Error::PodConversion` directly?